_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Bringing Serverless to a webpage near you with Hugo & Kubernetes"
+description: With the simplicity of OpenFaaS's templating system we can easily create new templates in no time. Today we will be showing how to continuosly deploy static sites using a new template for hugo and the FunctionIngress CRD for custom domain names.
+date: 2019-08-06
+image: /images/2019-serverless-static-sites/background.jpg
+categories:
+  - community
+  - end-user
+  - ingress
+  - kubernetes
+  - hugo
+author_staff_member: matias
+dark_background: true
+---
+
+Learn how you can continuosly deploy static sites with Hugo, OpenFaaS Cloud (or GitLab) and the FunctionIngress CRD.
+
+## Pre-req
+* Kubernetes
+
+You need to have a Kubernetes cluster up & running. I recommend [DigitalOcean](https://www.digitalocean.com/), it's cost-efficient and very easy to set-up.
```

Comment:
"I would recommend"

Add a link to the free credit that I have in my other post?